module/observationSubmissions/helper.js

```diff
@@ -1000,13 +1010,19 @@ module.exports = class ObservationSubmissionsHelper {
           entityIds.push(submission.entityId);
         });
 
-        let entitiesData = await entitiesHelper.entityDocuments(
+        let entitiesDocumentAPICall = await entityManagementService.entityDocuments(
```

Comment:
@borkarsaish65 rename this to entitiesDetails

Reply:
done

Reply:
@borkarsaish65 please change every where it is used

Reply:
done